src/python/runGeoShapeBenches.py

```diff
@@ -0,0 +1,268 @@
+import sys
+import pickle
+import subprocess
+import os
+import re
+import datetime
+import time
+
+#src/python/runGeoShapeBenches.py -compare -reindex -ant
+
+reTotHits = re.compile('totHits=(\d+)$')
+
+GEO_LOGS_DIR = '/data/geo'
+
+####################
+# Add here your paths
+####################
+GEO_UTIL_DIR ='/Users/ivera/forks/luceneutil_cleanfork'
+GEO_LUCENE_DIR = '/Users/ivera/forks/lucene-solr-fork/lucene'
+BASELINE_LUCENE_DIR = '/Users/ivera/projects/lucene-solr/lucene'
+#######
+# add your file name, it needs to be under your data directory (see IndexAndSearchShapes.java)
+#######
+fileName = "osmdata.wkt"
+
+shapes = ('point', 'box', 'poly 10', 'polyMedium', 'polyRussia')
+ops = ('within', 'disjoint', 'intersects')
+
+compareRun = '-compare' in sys.argv
+
+def printResults(results, stats, maxDoc):
```

Comment:
Should we also make a `.py` to write nightly charts, and then invoke this in the nightly cron job so we see progression over time?